store/sqlstore/post_store.go

```diff
@@ -991,13 +991,17 @@ func (s *SqlPostStore) AnalyticsUserCountsWithPostsByDay(teamId string) (model.A
 	return rows, nil
 }
 
-func (s *SqlPostStore) AnalyticsPostCountsByDay(teamId string) (model.AnalyticsRows, *model.AppError) {
+func (s *SqlPostStore) AnalyticsPostCountsByDay(teamId string, botsOnly bool, yesterdayOnly bool) (model.AnalyticsRows, *model.AppError) {
 	query :=
 		`SELECT
 		        DATE(FROM_UNIXTIME(Posts.CreateAt / 1000)) AS Name,
 		        COUNT(Posts.Id) AS Value
 		    FROM Posts`
 
+	if botsOnly {
+		query += " INNER JOIN Bots ON Posts.UserId = Bots.Userid"
```

Comment:
Might we refactor to leverage the squirrel library? (0/5, since this is pretty tidy already.)

Reply:
I'm also 0/5.  I like the idea of using an ORM, but see that it is hit or miss whether a designer uses it and find it easier to look at the actual SQL instead of ORM statements.  

Looking back at this code, maybe the squirrel library could help consolidate some of the `if POSTGRES` `else MySQL` statements, even in this function.  

Going to leave as is unless you or @crspeller say otherwise :)

Reply:
No objections to leaving as-is for this case, but let me just note that as per https://github.com/Masterminds/squirrel:
> Squirrel is not an ORM.

In general, I'm opposed to all things ORM, in part for the reason you cite -- obscuring the actual SQL. But I think squirrel fills a useful middle ground in dynamically constructed SQL statements, and does a decent job at /not/ obscuring same.

Anyway, I digress... :D

Reply:
0/5 this isn't too complicated. But if you want to learn squirrel go for it.